lib/SIL/IR/SILPrinter.cpp

```diff
@@ -3387,6 +3387,7 @@ void SILFunction::print(SILPrintContext &PrintCtx) const {
 
   switch (isThunk()) {
   case IsNotThunk: break;
+  case IsBackDeployedThunk: // FIXME: Give this a distinct label
```

Comment:
I will do this in a subsequent PR. I didn't want the test churn because this change needs to be cherry-picked.